src/index.ts

```diff
@@ -72,3 +73,4 @@ export { IgcInputBaseComponent } from './components/input/input-base.js';
 export { IgcSliderBaseComponent } from './components/slider/slider-base.js';
 export { IgcButtonBaseComponent } from './components/button/button-base.js';
 export { IgcCalendarBaseComponent } from './components/calendar/common/calendar-base.js';
+export { IgcMaskInputBaseComponent } from './components/mask-input/mask-input-base.js';
```

Comment:
That's kind of odd we have to do this, but sure

Reply:
👍 
Don't export it. 
I would vouch to remove the export of the other base classes as well in the future.